Tests/RestKitTests/JSONValueTests.swift

```diff
@@ -121,6 +136,18 @@ class JSONTests: XCTestCase {
         XCTAssertEqual(json.sorted(), expected.sorted())
     }
 
+    func testEncodeDate() {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
+        let expected = "2018-05-17T18:45:32.189Z"
+        let date = dateFormatter.date(from: expected)
+
+        let object: [String: JSON] = ["key": .date(date!)]
+        let data = try! JSON.encoder().encode(object)
```

Comment:
Likewise, avoid `try!` and handle failure properly.

Reply:
Will do.